lib/std/time/epoch.zig

```diff
@@ -133,7 +133,7 @@ pub const MonthAndDay = struct {
     day_index: u5, // days into the month (0 to 30)
 };
 
-// days since epoch Oct 1, 1970
```

Comment:
how the hell did October end up there? lol

Reply:
The best explanation I have is that someone misread `1970-01-01` as `1970-10-01` (which is why I did the commit message like that), but your guess is as good as mine :^)